packages/core/src/Effect/FiberState/index.ts

```diff
@@ -1,12 +1,12 @@
 import type * as Tp from "../../Collections/Immutable/Tuple"
-import type { Has, ServiceConstructor, Tag, TypeTag } from "../../Has"
+import type { Has, Tag, TypeTag } from "../../Has"
 import { tag } from "../../Has"
 import * as T from ".."
 import * as FRef from "../FiberRef"
 import * as L from "../Layer"
 
 export interface FiberState<S> {
-  readonly serviceId: `@effect-ts/core/Effect/FiberState<${TypeTag<S>}>`
+  //readonly serviceId: `@effect-ts/core/Effect/FiberState<${TypeTag<S>}>`
```

Comment:
@mikearnaldi I wasn't sure how to do this one with TypeTag, from `makeFiberState`